easybuild/easyconfigs/s/scanpy/scanpy-1.9.1-foss-2022a.eb

```diff
@@ -0,0 +1,76 @@
+# updated: Denis Kristak (INUITS)
+easyblock = 'PythonBundle'
+
+name = 'scanpy'
+version = '1.9.1'
+
+homepage = 'https://scanpy.readthedocs.io/en/stable/'
+description = """Scanpy is a scalable toolkit for analyzing single-cell gene expression data built
+ jointly with anndata. It includes preprocessing, visualization, clustering, trajectory inference
+ and differential expression testing. The Python-based implementation efficiently deals with
+ datasets of more than one million cells.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'openmp': True}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('h5py', '3.7.0'),
+    ('networkx', '2.8.4'),
+    ('numba', '0.56.4'),
+    ('PyTables', '3.8.0'),
+    ('statsmodels', '0.13.1'),
+    ('scikit-learn', '1.1.2'),
+    ('Seaborn', '0.12.1'),
+    ('tqdm', '4.64.0'),
+    ('leidenalg', '0.9.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('natsort', '8.2.0', {
+        'checksums': ['57f85b72c688b09e053cdac302dd5b5b53df5f73ae20b4874fcbffd8bf783d11'],
+    }),
+    ('xlrd', '2.0.1', {
+        'checksums': ['f72f148f54442c6b056bf931dbc34f986fd0c3b0b6b5a58d013c9aef274d0c88'],
+    }),
+    ('anndata', '0.8.0', {
+        'checksums': ['94d2cc6f76c0317c0ac28564e3092b313b7ad19c737d66701961f3e620b9066e'],
+    }),
+    ('pynndescent', '0.5.8', {
+        'checksums': ['a7c552569bf604a101fd54bba1d27c12389e065945dee3a6777a518c63a46f2b'],
+    }),
+    ('umap-learn', '0.5.3', {
+        'modulename': 'umap',
+        'checksums': ['dbd57cb181c2b66d238acb5635697526bf24c798082daed0cf9b87f6a3a6c0c7'],
+    }),
```

Comment:
depend on the `umap-learn` easyconfig instead

Reply:
solved